website_rentals/models/sale_order_line.py

```diff
@@ -0,0 +1,20 @@
+from odoo import models, fields
+import datetime
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    pickup_date_no_timezone = fields.Char(string="Pickup", compute='_compute_pickup_date_no_timezone')
```

Comment:
Change the string or it triggers warnings on SH builds for nothing:
> Two fields (pickup_date_no_timezone, pickup_date) of sale.order.line() have the same label: Pickup.

E.g "Pickup date without timezone"